File: _java/ch04/src/ch15/sec05/exam03/Person.java
package ch15.sec05.exam03;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Person {
    public String name;
    public int age;

    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }




}

File: _java/java_basic/src/main/kotlin/ch16/sec05/exam03/Person.kt
package ch16.sec05.exam03

class Person {
     fun getMember1(creatable: Creatable1): Member? {
        val id = "winter"
        val member = creatable.create(id)
        return member
    }

    fun getMember2(creatable: Creatable2): Member? {
        val id = "winter"
        val name = "한겨울"
        val member = creatable.create(id, name)
        return member
    }
}